src/components/totals/FundsRaised/index.js

```diff
@@ -38,6 +39,7 @@ module.exports = React.createClass({
       startAt: null,
       endAt: null,
       offset: 0,
+      onLoad: function () {},
```

Comment:
Minor nitpick but I don't think you need a default prop here – since you're checking the type on line 103